lib/experimental/new_embed.dart

```diff
@@ -803,6 +811,11 @@ major browsers, such as Firefox, Edge (dev channel), or Chrome.
     return url.queryParameters['theme'] == 'dark';
   }
 
+  bool get autoRunEnabled {
```

Comment:
Not entirely your code, but since you're adding a new URL param getter, how about moving this group up to the section in 400-430? Right now we have two groups in different places.